os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -89,7 +89,7 @@ static u8 lcdc_nextframe = 0;
 struct irq lcdc_irq_info = {
 	.num = LCDC_IRQ,
 	.data = (u32) LCDC,
-	.priority = INT_PRI_MIDDLE,
+	.priority = INT_PRI_HIGH,
 };
 
 extern struct irq mipi_irq_info;
```

Comment:
Do we need to increase the priority of mipi also? It is currently set to middle.

Reply:
MIPI irq is only used during sending init command table. When running LCD app, only LCD irq is being applied.